### ISSUE TITLE
RejectedAccessException of docker from Template in 2.0.3

### ISSUE THREAD

**Issue by cokieffebah** [jenkinsci/templating-engine-plugin#171]
### Environment Information

- [ ] Jenkins version
- [ ] Plugin version: 2.0.3
- [ ] OS: Docker Jenkins

### Description

Please describe your issue here.

got the following
org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: Scripts not permitted to use method org.jenkinsci.plugins.docker.workflow.Docker image java.lang.String

after adding the following to a previously working template in a multibranch job
```
docker.image("intoto-porter:demo").inside {
        unstash workspace
        sh("porter create")
        deploy()
        stash workspace
}
```

**Comment by cokieffebah** [comment]
This block, excluded the declaration of the image, was copied from a library step that is working

**Comment by steven-terrana** [comment]
I'm not seeing anything here that indicates a bug in JTE unless our `SandboxWhitelist` class is somehow triggering this exception. 

Can you try running the same code in a regular sandboxed pipeline job and see if you still get the exception?

**Comment by cokieffebah** [comment]
Ran in Pipeline job and succeeded though I got a script approval 'message', not a rejection.
<img width="1079" alt="Screen Shot 2021-03-01 at 1 17 33 PM" src="https://user-images.githubusercontent.com/23703182/109540547-9c3c2a00-7a90-11eb-850e-95fc25aab16a.png">

**Comment by steven-terrana** [comment]
i haven't been able to reproduce this one locally yet. 🤔

**Comment by cokieffebah** [comment]
updated script security plugin, no change.  will try to update other plugins, later

**Comment by cokieffebah** [comment]
Resolved in 2.1
